testing/upload_validation.ts

```diff
@@ -421,20 +436,10 @@ const commonPackFormulaSchema = {
   extraOAuthScopes: z.array(z.string()).optional(),
 };
 
-const numericPackFormulaSchema = zodCompleteObject<Omit<NumericPackFormula<any>, 'execute'>>({
-  ...commonPackFormulaSchema,
-  resultType: zodDiscriminant(Type.number),
-  schema: zodCompleteObject<NumberSchema>({
-    type: zodDiscriminant(ValueType.Number),
-    codaType: z.enum([...NumberHintValueTypes]).optional(),
-    description: z.string().optional(),
-  }).optional(),
-});
-
 const stringPackFormulaSchema = zodCompleteObject<Omit<StringPackFormula<any>, 'execute'>>({
   ...commonPackFormulaSchema,
   resultType: zodDiscriminant(Type.string),
-  schema: zodCompleteObject<StringSchema>({
+  schema: zodCompleteObject<BaseStringSchema>({
```

Comment:
This needs to be using stringPropertySchema, for the same reason that you changed numericPackFormulaSchema to use numberPropertySchema, right?

Reply:
ugh, sorry, missed that too. I think I did a find-replace in vscode and it overeagerly renamed a bunch of things.

Reply:
I think you just need to address this one, and then we're good.

Reply:
ah good catch! updated!